project-docs/GOVERNANCE.md

```diff
@@ -0,0 +1,63 @@
+# Vega Project Governance
+
+This is the default project governance for the Vega Projects. Our goal is to encourage a diverse community of contributors and users to participate in the development of the Vega projects.
+
+Each project may override the specific roles and development guidelines in a `GOVERNANCE.md` and `CONTRIBUTING.md` file. However, all projects have to adhere to the code of conduct, the general principles outlined in this document, and the charter.
+
+## Roles
+
+We define three levels of roles in the Vega projects: Contributors, Maintainers, and Admins. We expect everyone to adhere to our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+### Contributors
+
+Contributors are anyone who makes a contribution to a Vega project via GitHub. This includes filing issues, sending pull requests, and participating in discussions. There are no requirements to participate and we encourage everyone to contribute. You do not have to be a Maintainer to send a pull request.
+
+### Maintainers
+
+Maintainers belong to all or some subset of Vega projects. Maintainers are responsible for organizing activities around developing, maintaining, and updating the project. Maintainers are also responsible for determining consensus.
+
+Maintainers have write access and can push directly to branches on GitHub. Maintainers also have access to specific developer channels on Slack.
+
+To become a Maintainer, a contributor must be nominated by an existing Maintainer and approved by a simple majority of Maintainers via Slack reactions within 7 days (simple majority of responses). A new Maintainer should be added to 1) the GitHub team, 2) the Slack channel, and 3) the [MAINTAINERS.md file](MAINTAINERS.md). We want more people to become Maintainers and lower the barrier the entry. If you are interested in becoming a Maintainer, please reach out to an existing Maintainer.
```

Comment:
Just want to double check a couple of things here:

* Do these two lines mean that we'll be switching the `-dev` channels from public to private on the Vega slack?
* Will votes for maintainers (and actually any membership level) occur on these private channels? If so, there is some potential for awkward power dynamics—eg person A is nominated; there is an up/down vote, they're ultimately added as a Maintainer and join the private channel and can now see who voted against their membership. I imagine this will be a relatively rare event, so I don't think we need to overengineer it. I'm also on the fence about how much of an issue this actually is. But, one lightweight approach might be for whomever is chairing the nomination process to announce the result at the conclusion of the vote (eg as number for/against), and then delete the Slack post corresponding to the vote.

Reply:
For channels, I made new ones just to now kick people out of channels but we can basically "archive" the existing ones at some point. 

Hm, right, I hadn't thought about the awkwardness of being able to see who voted against you. @jheer suggested in the steering committee meeting that we could use Google forms for anonymous voting. We could switch to that for votes on people. I would suggest that we restrict access to the results to the chairs and they should delete the form after the vote has passed similar to what you suggest for Slack votes. 

I'd say for now let's do what you suggested and delete the vote post after the vote.